doc/release-notes/iceoryx2-unreleased.md

```diff
@@ -17,6 +17,9 @@
 
  * Fix retrieve channel overflow caused by big publisher loans [#116](https://github.com/eclipse-iceoryx/iceoryx2/issues/116)
  * Fix `open_or_create` race [#108](https://github.com/eclipse-iceoryx/iceoryx2/issues/108)
+ * Fix `CreationMode::OpenOrCreate` in `iceoryx2-bb-posix::SharedMemory` [#116](https://github.com/eclipse-iceoryx/iceoryx2/issues/116)
+ * Add missing memory synchronization to posix shm zero copy connection [#116](https://github.com/eclipse-iceoryx/iceoryx2/issues/116)
+ * Remove retrieve buffer full check from zero copy connection - sender had insufficient infos available [#116](https://github.com/eclipse-iceoryx/iceoryx2/issues/116)
```

Comment:
What do you think about grouping like
```
* Fixes for [#42](...)
    * Fix ...
    * Add ...
    * Remove ...
```

Reply:
Yes, I like it. I will do this in the future.